pkg/ui/static/js/graph_template.handlebar

```diff
@@ -80,14 +74,14 @@
                         <div class="prometheus_input_group pull-left">
 
                           <button
-                              class="btn btn-default pull-left"
+                              class="btn btn-light pull-left"
                               type="button"
                               name="dec_end"
                               title="Rewind the end time.">
                             <i class="glyphicon glyphicon-backward"></i>
-                          </button>
+                          </button><!--
```

Comment:
do we need this comment, for example? that's why I'm wondering 😄

Reply:
Just following the upstream (:  I don't know why it was not deleted by Prometheus too. ):